src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  const isSelectedTab = tabId => selectedTab.id === tabId;
```

Comment:
let's move this into `.map` block, so you don't need to call this function every time when you need